projects/ngneat/transloco-messageformat/src/lib/messageformat.module.ts

```diff
@@ -0,0 +1,33 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { TRANSLOCO_TRANSPILER } from '@ngneat/transloco';
+import { TranslocoMessageFormat } from './messageformat.transpiler';
+import { TRANSLOCO_MESSAGE_FORMAT_CONFIG, MessageformatConfig } from './messageformat.config';
+
+export function translocoMessageFormatFactory(config: MessageformatConfig) {
+  return new TranslocoMessageFormat(config);
+}
+
+@NgModule({
+  providers: [
```

Comment:
Why duplication? They should always call `init`.

Reply:
I wanted to show that you don't necessarily need to call the init function

Reply:
We should be consistent.